doc/developer_guide.md

```diff
@@ -172,24 +172,41 @@ You can embed the Extension Manager's REST API in other programs that use the [N
 ```go
 // Create an instance of `openapi.API`
 api := openapi.NewOpenAPI()
-// Create a new configuration object
-config := ExtensionManagerConfig{ExtensionRegistryURL: "https://<extension-registry>"}
+// Create a new configuration
+config := extensionController.ExtensionManagerConfig{
```

Comment:
In discovered the term config often to be a synonym for "util", "help", or "anything".
Would "endpoints" be a more specific name for this variable and also for the struct? 
Or do you expect future enhancements?